### PR TITLE
Math: Covert to ES6 default value.

### PR DESCRIPTION
**Pull request by linbingquan**
**Description**

Math: Covert to ES6 default value.

### CODE REVIEW

File: src/math/Matrix4.js [hunk 1]
```diff
@@ -822,9 +822,7 @@ class Matrix4 {
 
 	}
 
-	fromArray( array, offset ) {
-
-		if ( offset === undefined ) offset = 0;
+	fromArray( array = [], offset = 0 ) {
```
Comment (Mugen87) [inline src/math/Matrix4.js:825]: The `array` parameter should never be optional when using `fromArray()`.

Comment (linbingquan) [reply src/math/Matrix4.js]: Thanks for your review, I corrected it.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

File: src/math/Matrix4.js (hunk 1 above)
Comment (WestLangley) [reply src/math/Matrix4.js]: @Mugen87 I do not think the array parameter should be optional when using `.toArray()` either. It is supposed to be populating an existing array, not instantiating one. Instantiating one violates the three.js policy.

**Pull request merged**